Mage.Sets/src/mage/cards/s/SlivdraziMonstrosity.java

```diff
@@ -0,0 +1,109 @@
+package mage.cards.s;
+
+import mage.MageInt;
+import mage.ObjectColor;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.effects.common.continuous.BecomesSubtypeAllEffect;
+import mage.abilities.effects.common.continuous.GainAbilityControlledEffect;
+import mage.abilities.keyword.AnnihilatorAbility;
+import mage.abilities.keyword.DevoidAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterPermanent;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.game.permanent.token.EldraziSliverToken;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+public final class SlivdraziMonstrosity extends CardImpl {
```

Comment:
Don't forget to add your name here since you're the author.

Reply:
I've been leaving those out because I personally find them somewhat pointless (especially since IntelliJ says who last touched each line of code if you care...) but done.